doc/spec/api.rst

```diff
@@ -164,6 +173,23 @@ Given the prevalence of linear treatment effect assumptions, we will create a ge
 Example Use of API
 ------------------
 
+Let us walk through a simple example of what one can achieve via the latter API
+even irrespective of the actual estimation method that is being used.
+
+Let us consider a hypothetical data generating process governed by the 
+following equations:
+
+.. math ::
+    \begin{align}
+        Y(t) =~& \gamma t^2 + \delta X t + \langle \zeta, W \rangle + \epsilon\\
```

Comment:
~ add an extra space in math. It looks better. Only git does not parse this correctly. Sphinx does. We should keep the ~.

Reply:
I removed it from the other place because the CATE and marginal CATE words where going off the margin and so I removed the extra space to make them fit within the white margin,

Reply:
Ah, ok, sounds good.